homeassistant/components/apple_tv/media_player.py

```diff
@@ -110,61 +115,44 @@ def should_poll(self):
     @property
     def state(self):
         """Return the state of the device."""
-        if not self._power.turned_on:
+        if self._manager.is_connecting:
+            return STATE_UNKNOWN
```

Comment:
Use `None` to represent unknown state.
```suggestion
            return None
```